controls/pcidss_4_ocp4.yml

```diff
@@ -2144,10 +2199,11 @@ controls:
         interception of communications or from stored data.
       levels:
         - base
-      status: pending
+      status: not applicable
```

Comment:
wondering if `ocp_no_ldap_insecure` also applies to here, since it is related to transmission of password